### PR TITLE
Fix theme preview image scale and margin problem

### LINKED ISSUE #26385: Theme selector preview image fitting problem BO/multistore
#### Describe the bug

BO multistore settings, when add new shop, under the theme selector, the preview image won't fit. Missing max-width:100%, and there is set a margin-bottom for some reason.

#### Expected behavior
kép
kép

#### Steps to Reproduce

1. Enable multistore
2. Add new shop

kép

#### Additional information

* PrestaShop version: 1.7.8.0
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by SNor42**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      |  It can be found under BO/Multishop/Shop details. It fix the theme preview image scale and margin problem.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #26385 .
| How to test?      | SCSS need to compile to see changes.
| Possible impacts? | Guess nothing

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26459)

### CODE REVIEW

**Comment by NeOMakinG** [comment]
@SNor42 we need to wait for the UX validation for the button style

If it's validated (on the issue), we also need to do it inside the UIKit and create a release + update the UIKit on PrestaShop

**Comment by NeOMakinG** [comment]
@SNor42 Scott commented on the issue, it looks like we don't want to change switches, but the https://github.com/PrestaShop/PrestaShop/issues/26385 fix remain needed :)

File: admin-dev/themes/default/scss/partials/_switch-btn.scss [hunk 1]
```diff
@@ -103,7 +103,7 @@
       }
 
       ~ .slide-button {
-        background: $brand-success;
+        background: $primary-color;
```
Comment (alexandrebru83) [inline admin-dev/themes/default/scss/partials/_switch-btn.scss]:
```suggestion
        background: $brand-success;
```

Can you commit this change ? 
Like @NeOMakinG said "it looks like we don't want to change switches"

Your PR will be approved to fix the issue about preview image theme

**Review by alexandrebru83 — CHANGES_REQUESTED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
@NeOMakinG should we close this PR ?

**Review by NeOMakinG — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
@matthieu-rolland no, one issue remains fixed in it :)

**Review by alexandrebru83 — DISMISSED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
There's a linter issue it's blocking the CI, once it's fixed it's lgtm 👍

**Comment by NeOMakinG** [comment]
@SNor42 could you pull, run `npm run scss-fix` and push please? :)

**Review by matthieu-rolland — COMMENTED** [review]
you should also rebase your PR, it will fix the phpstan issue

**Comment by SNor42** [comment]
Sorry Guys!
I'm noob on GitHub, will ask google. But maybe it's better if some pro will
do those steps.

Matthieu Rolland ***@***.***> ezt írta (időpont: 2021. okt.
29., P, 15:09):

File: admin-dev/themes/default/scss/controllers/_themes.scss [hunk 2]
```diff
@@ -19,6 +19,11 @@
     border-bottom: solid 1px #fff;
   }
 
+  .thumbnail {
+    max-width: 100%;
+    margin-bottom: 0px;
```
Comment (Progi1984) [inline admin-dev/themes/default/scss/controllers/_themes.scss]:
```suggestion
    margin-bottom: 0;
```

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by florine2623** [comment]
Hello @SNor42 ,

Thanks for the PR !

With 2 templates added : OK
Screenshot 2021-11-03 at 12 08 49

When I add multiple templates with different lengths, the design is not optimal :
Screenshot 2021-11-03 at 12 19 00

Could you take a look ?
Thanks!

**Comment by florine2623** [comment]
Hello @SNor42 ,

Have you had the time to check this ? :)
It would be really great to be able to have a clean look when more than 4 templates are added !

Here's a mockup created by @prestascott , you can take inspiration from that :
Capture d’écran 2021-11-17 à 16 30 55

Let me know if you have any questions.
Thanks!

**Review by Progi1984 — CHANGES_REQUESTED** [review]
@SNor42 Hi, Could you remove your merge commits ?

**Comment by SNor42** [comment]
@Progi1984: Sure, but i'm noob. How?

**Comment by NeOMakinG** [comment]
@SNor42 I think if you do `git rebase upstream/develop` on your branch and `git push origin develop -f`, everything should be fine!

But you need to have the official repo as a `upstream` named remote on your fork

`git add remote upstream git@github.com:PrestaShop/PrestaShop.git`
`git fetch upstream`

**Comment by SNor42** [comment]
Hi!
Thanks. I've tried and got these messages:
1) Current branch develop is up to date.
2) Everything up-to-date.
Are these okay?

**Comment by SNor42** [comment]
Hi!
I've tried. If it's not working, I could remove my repository. If that's help.

**Comment by NeOMakinG** [comment]
@SNor42 I did the rebase, but something went wrong and your commit disappeared.

I tried to add your work again, but I can't push to your fork 

I think it would be way easier if you created another branch before creating the PR :/

Could you push your changes again, but maybe on another branch? And change the branch of the PR! 

Tried to help but damn I was surprised that after the rebase, your commit disappeared...

**Pull request closed without merging**